### ISSUE TITLE
Clarify iontestdata/bad/minLongWithLenTooSmall.10n

### ISSUE THREAD

**Issue by PeytonT** [amzn/ion-tests#60]
The [description of the test](https://github.com/amzn/ion-tests/blob/master/iontestdata/bad/README.md#minlongwithlentoosmall10n) from the README seems to be wrong, and the binary specification seems to suggest that the test case should parse as valid.

_minLongWithLenTooSmall.10n_
--------------------------
_Contains an Int whose length is specified as 7 bytes, but contains 8 bytes of
data. The trailing byte is `0x00` (a Null with an invalid L value)._

The [binary specification](http://amzn.github.io/ion-docs/docs/binary.html#0-null) describes `0x00` as a single byte NOP pad, and defines NOP padding as valid anywhere a value can be encoded, except for within an annotation wrapper. It should be valid to have an arbitrarily number of `0x00` trailing a value.

**Comment by PeytonT** [comment]
To add to this, the bad test README describes a [nullBadTD.10n](https://github.com/amzn/ion-tests/tree/master/iontestdata/bad#nullbadtd10n) test case that doesn't seem to exist.

**Comment by jonwilsdon** [comment]
Thanks for creating this issue!

The actual file contents are:
```
E0 01 00 EA 37 80 00 00 00 00 00 00 01  
^.......... ^..1..2..3..4..5..6..7  ^..2?
BVM         7 byte int              2 byte NOP pad
```
The text in the README should be updated to specify the correct trailing byte (`0x01`) and that the file is invalid because it is missing the 2nd NOP pad byte.

**Comment by tgregg** [comment]
We should delete the nullBadTD.10n section from the README. The file was moved to `good/` when NOP padding was added to the spec, but the README entry remained.

**Comment by PeytonT** [comment]
@jonwilsdon 
Great to know! I had been pondering why the official parsers raise parse errors on this file if `0x00` is a valid trailing byte, and I was wondering if I was going to have to hunt down a bug in my parser (which raises a parse error for early EOF). I suppose I should have gone and taken a peek at the last byte!

**Comment by tgregg** [comment]
Closed via #62 .
